cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -257,6 +257,17 @@ void Run(const std::vector<string>& bag_filenames) {
     }
 
     bag.close();
+
+    if (FLAGS_keep_running) {
```

Comment:
I just rechecked - it seems that my University's mail system has not forwarded the message to my GMail account. This is quite alarming, I now have to recheck if I have missed any other mails :O Today seems good, see you then!